extractors/communication/pigeon/index.md

```diff
@@ -12,9 +12,8 @@ being sent or received. When running, the extractor will import all emails recei
 Therefore it is a good idea to set up Pigeon Importer in a [**scheduled** orchestration](/orchestrator/scheduling/). 
 
 ## Create New Configuration
-Find Pigeon Importer in the Extractors section and create a new configuration. 
-As each configuration can import multiple tables, there is no need to create more than one unless 
-you want to better organize your data. Let's say one configuration would load your daily data, and
+Find Pigeon in the Extractors section and create a new configuration. 
+One configuration can only import one table. Let's say one configuration would load your daily data, and
```

Comment:
tohle me ted moc nedava smysl, predtim mi to prislo lepsi

Reply:
Myslis to chovani toho extractoru?

Reply:
ne, na co potrebujes vic konfiguraci

Reply:
Ok, vratim puvodni text.